games-board/lc0/lc0-0.21.1.ebuild

```diff
@@ -0,0 +1,222 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils flag-o-matic meson toolchain-funcs unpacker
+
+MY_MKL_PV="10.0.5.025"
+
+DESCRIPTION="UCI-compliant chess engine designed to play chess via neural network"
+HOMEPAGE="https://lczero.org/"
+
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64"
+IUSE="amd-gpupro-opencl amd-opencl +blas-ext blas-int mkl nvidia-cudnn nvidia-opencl switch"
+REQUIRED_USE="?? ( amd-gpupro-opencl amd-opencl )
+	      ?? ( blas-ext blas-int )
+	      || ( amd-gpupro-opencl amd-opencl blas-ext blas-int mkl nvidia-cudnn nvidia-opencl )
+	      amd-gpupro-opencl? ( !nvidia-cudnn !nvidia-opencl )
+	      amd-opencl? ( !nvidia-cudnn !nvidia-opencl )"
+
+# Build needs extra submodule.
+MY_MODULE_N="proto"
+MY_MODULE_URI="https://github.com/LeelaChessZero/lczero-common/archive/master.tar.gz"
+
+# We need network weight for runtime.
+MY_W_ID="50778"
+MY_W_N="weights"
+MY_W_SHA="5b535e00f1828556b9199ce7d4dbdbbf91cc8c2f9084d991f9f4ec7d43baa6e5"
+MY_W_URI="https://lczero.org/get_network?sha=${MY_W_SHA}"
+
+# We need openblas-dev libraries if we don't have internally.
+MY_BLAS_BASE_N="libopenblas-base"
+MY_BLAS_DEV_N="libopenblas-dev"
+MY_BLAS_PV="0.3.5"
+MY_BLAS_SRC="http://archive.ubuntu.com/ubuntu/pool/universe/o/openblas"
+MY_BLAS_BASE_URI="${MY_BLAS_SRC}/${MY_BLAS_BASE_N}_${MY_BLAS_PV}+ds-2_amd64.deb"
+MY_BLAS_DEV_URI="${MY_BLAS_SRC}/${MY_BLAS_DEV_N}_${MY_BLAS_PV}+ds-2_amd64.deb"
```

Comment:
As discussed on IRC, lets drop this openblas stuff here. and make a separate ebuild in the tree. It'll be a lot cleaner overall.

Reply:
Done.
Separate PR for 'openblas' -> https://github.com/gentoo/gentoo/pull/11700